openapi/api_trial_system.template.yaml

```diff
@@ -0,0 +1,120 @@
+swagger: "2.0"
+info:
+  version: 0.0.1
+  title: Trial System API
+host: localhost
+basePath: /trials/api/v1
+schemes:
+  - https
+security:
+  - Bearer: []
+securityDefinitions:
+  Bearer:
+    type: apiKey
+    name: Authorization
+    in: header
+paths:
+  "/subscriptions":
+    get:
+      operationId: getSubscription
+      summary: Get subscription detail
+      description: >-
+        Returns the details about the subscription of the authenticated user for
+        the specified trial.
+      parameters:
+        - $ref: "#/parameters/queryTrialId"
+      responses:
+        "200":
+          $ref: "#/responses/200Subscription"
+        "401":
+          $ref: "#/responses/401Unauthorized"
+        "429":
+          $ref: "#/responses/429TooManyRequest"
+        "500":
+          $ref: "#/responses/500Error"
```

Comment:
I used `$ref` to remove the duplication of the description and for clarity, but I'm not sure the generator handles this syntax.

Reply:
**UPDATE**: Using reference on response the client generates them using `unknown` type. For now I've removed the reference.